### PR TITLE
Bump up size for estimator_test

### PR DESCRIPTION
**Pull request by terrytangyuan**
Avoid estimator_test timeout as seen in other PR builds, such as https://ci.tensorflow.org/job/tensorflow-pull-requests-cpu/3437/console

### CODE REVIEW

**Comment by gunan** [comment]
@martinwicke This change will disable this test in some of our CI builds.
I think the change is warranted right now. But we will eventually need to resolve it.

**Comment by caisq** [comment]
@gunan @martinwicke @terrytangyuan estimator_test is currently not sharded. Is it possible to shard it, like `shard_count = 4`?

**Comment by terrytangyuan** [comment]
@caisq Good idea. I've changed the commit to add shard count, assuming all tests are independent.

**Comment by terrytangyuan** [comment]
@gunan Just for my curiosity. Large tests are disabled internally? Under what environments/devices?

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

File: tensorflow/contrib/learn/BUILD [hunk 1]
```diff
@@ -478,6 +478,7 @@ py_test(
 py_test(
     name = "estimator_test",
     size = "medium",
+    shard_count = 4,
```
Comment (rmlarsen) [inline tensorflow/contrib/learn/BUILD]: this breaks the build. Should this be a tf_py_test to support shard_count?

**Review by rmlarsen — DISMISSED** [review]

File: tensorflow/contrib/learn/BUILD (hunk 1 above)
Comment (caisq) [inline tensorflow/contrib/learn/BUILD]: +1

Comment (terrytangyuan) [inline tensorflow/contrib/learn/BUILD]: Why is that? All other tests that use shard_count seem to be py_test

**Comment by caisq** [comment]
@terrytangyuan You are right. The test failures may be due to a BUILD file format requirement. Can you move the `shard_count` line after `srcs` and before `srcs_version`.

**Comment by gunan** [comment]
No, please do not use sharding, they can cause even worse complications
with testinfra. Id rather we just disable this test.

Large tests have 15 minute timeouts, and build phase added on top of that
takes them out of our goals for presubmit time limits. Therefore we
conciously disable large tests in our testinfra.

**Comment by terrytangyuan** [comment]
Okay...should I change this to manual? Let me know what's the correct way
to do it. I think it's better to document the best practices somewhere to
avoid this back and forth conversation.

**Comment by martinwicke** [comment]
The best thing to do would be to split this test into more manageable
pieces, or even better, make it faster (less iterations, etc., if such a
thing is still possible).

On Wed, Feb 1, 2017 at 10:56 AM, Yuan (Terry) Tang <notifications@github.com
> wrote:

> Okay...should I change this to manual? Let me know what's the correct way
> to do it. I think it's better to document the best practices somewhere to
> avoid this back and forth conversation.
>

**Comment by gunan** [comment]
Id prefer either Martin's suggestion, or disabling the test. I am OK with
either right now, as this is becoming a blocker for many contributions.

**Review by gunan — APPROVED** [review]

**Comment by terrytangyuan** [comment]
@tensorflow-jenkins test this please

**Comment by rmlarsen** [comment]
The failing tests are unrelated. Merging.

**Pull request merged**